dags/reporting-dag.py

```diff
@@ -46,21 +45,11 @@ def set_max_task_instance(ti, **kwargs):
     ti.xcom_push(key='MAX_TASK_START', value=str(max_task_start[0]))
 
 
-default_args = {
-    'owner': 'airflow',
-    'depends_on_past': False,
-    'email_on_failure': False,
-    'email_on_retry': False,
-    'retries': 1,
-    'retry_delay': timedelta(minutes=5)
-}
-
 with DAG('reporting_dag',
```

Comment:
Would prefer if the filename and DAG name matched, right now you have mixed `-` and `_`.

Reply:
done